phoenix-core/src/it/java/org/apache/phoenix/end2end/CreateTableIT.java

```diff
@@ -1557,7 +1558,7 @@ public void testCoprocessorsForCreateIndexOnOldImplementation() throws Exception
             // Now roll back to the old indexing
             IndexUpgradeTool iut =
                     new IndexUpgradeTool(ROLLBACK_OP, tableName, null,
-                            "/tmp/index_upgrade_" + UUID.randomUUID().toString(), false, null,
+                            filePath + "/index_upgrade_" + UUID.randomUUID().toString(), false, null,
```

Comment:
In the Hadoop Tools, path parameters are some times HDFS paths, so that is a case where simple "/tmp" is the right solution.
Those are also separate HDFS file systems, so there is no danger of coliision.

Have you checked that all of these instances are used as local paths, as opposed to HadoopFS paths ?

(I have checked this one, and this is correct, as this is a local FS path)